ansible/test.yml

```diff
@@ -95,6 +95,21 @@
       that:
       - "sphinx_version.stdout == 'Sphinx (sphinx-build) 1.2.3'"
 
+- hosts: jekyll
+  roles:
+  - role: jekyll-site
+    jekyll_site_name: test
+    jekyll_site_git_repo: https://github.com/lunson/www.openmicroscopy.org
```

Comment:
The proper www.openmicroscopy.org repo would make more sense, but as far as I can tell there's not much there yet.